figures/scripts/fig4-hgg-subtype-forest-plot.R

```diff
@@ -177,4 +182,8 @@ forest_panels <- cowplot::plot_grid(forest_plot, labels_panel, nrow = 1, rel_wid
 ggsave(forest_pdf, forest_panels, width = 10, height = 3)
 
 
+# Export CSV for Zenodo upload
+# no samples so nothing to arrange
+readr::write_csv(survival_df_spread, fig4g_csv)
```

Comment:
I think we want `survival_df` here? That is what is used to make the plot, whereas the `spread` version is only for the labels, from what I see

Reply:
Yeah, I went back and forth for this actually.. This one matched the figure itself, whereas `survival_df` matches the data underlying the figure! I think there isn't necessarily a Best answer here, but I'll change this to the non-spread ones for both forest plots.